pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -723,6 +723,10 @@ type VerticalPodAutoscaler struct {
 	// (default: 0.95)
 	// +optional
 	RecommendationUpperBoundCPUPercentile *float64 `json:"recommendationUpperBoundCPUPercentile,omitempty" protobuf:"fixed64,11,opt,name=recommendationUpperBoundCPUPercentile"`
+	// CPUHistogramDecayHalfLife is the amount of time it takes a historical CPU usage sample to lose half of its weight.
+	// (default: 24h)
```

Comment:
Augment https://github.com/gardener/gardener/blob/24408b594d4028a4eae4d2691022888c2e7445e4/pkg/apis/core/v1beta1/defaults_shoot.go#L223-L277

Reply:
I was wondering a bit why we're doing an additional layer of defaulting – is it to protect ourselves from changes of the upstream defaults? Otherwise, these defaults could all be removed, as we're keeping them on the same values that are default in upstream VPA.

Reply:
At some point in time, we decided that we want to default the values directly in the `Shoot` spec (if there is a default value) instead of silently configuring the respective component. This approach allows the user to see what is going on instead of looking it up in some doc (that might not even exist), or - even worse - in the source code.
This might not be consistently implemented for all fields, yet we at least tried to do it for newly introduced fields.